service/debugger/debugger.go

```diff
@@ -629,6 +639,12 @@ func (d *Debugger) CreateBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoin
 		return nil, err
 	}
 
+	for _, addr := range addrs {
```

Comment:
Can you move this check inside createLogicalBreakpoint? In case we ever reuse it somewhere else we'll want this check in there.